libs/wingsdk/src/target-tf-azure/bucket.inflight.ts

```diff
@@ -34,6 +34,30 @@ export class BucketClient implements IBucketClient {
     );
   }
 
+  /**
+   * Check if an object exists in the bucket
+   *
+   * @param key Key of the object
+   */
+  public async exists(key: string): Promise<boolean> {
+    const connectionString =
+      "DefaultEndpointsProtocol=https;AccountName=myaccount;AccountKey=mykey;EndpointSuffix=core.windows.net";
+
+    const blobServiceClient =
+      BlobServiceClient.fromConnectionString(connectionString);
+    const containers = blobServiceClient.listContainers();
+    for await (const container of containers) {
+      const containerClient = blobServiceClient.getContainerClient(
+        container.name
+      );
+      const blobClient = containerClient.getBlobClient(key);
+      if (await blobClient.exists()) {
+        return true;
+      }
+    }
+    return false;
```

Comment:
I don't have an active Azure Subscription for trying it out. I guess it wouldn't work since you would need to edit the connection string to include your Azure account.

Reply:
It's OK, let's just omit the implementation for Azure for now :)
Can you please open an issue (for completing `Bucket.exists` method on Azure) 
so we'll know what is missing?

Reply:
then we'll just have the unit test (for tf-aws and sim) and the SDK spec test, and we're good to go!

Reply:
Ticket for finishing `tf-azure` implementation 🡆 https://github.com/winglang/wing/issues/2575

I'll add tests for `tf-aws` and `sim` soon.

Reply:
Thanks @garysassano! 
If the Azure code wasn't tested I would go ahead and delete it from this PR for now - it's not a lot of code and I believe testing will show us we need a few tweaks anyway.

Reply:
@staycoolcall911 If I remove the `exists` method from the `tf-azure` target, then the tests would fail because `BucketClient` needs to implement `IBucketClient` interface.

Reply:
You can implement and throw unsupported operation exception or something of that sort. We should have a few places with similar implementation. LMK if you need assistance locating one